src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -50,4 +59,32 @@ public void setColor(String color) {
     public List<Section> getSections() {
         return sections;
     }
+
+    public Section getLastSection() {
+        return this.getSections().get(this.getSections().size() - 1);
+    }
+
+    public void addSection(Section section) {
+        this.sections.add(section);
+    }
+
+    public void update(LineUpdateDto lineUpdateDto) {
+        if (lineUpdateDto.getName() != null) {
+            this.name = lineUpdateDto.getName();
+        }
+
+        if (lineUpdateDto.getColor() != null) {
+            this.color = lineUpdateDto.getColor();
+        }
+    }
+
+    public void removeSection(Section section) {
+        this.sections.remove(section);
+    }
+
+    public void validateBeforeRemoveSection(Station station) {
+        if (!this.getSections().get(this.getSections().size() - 1).getDownStation().equals(station)) {
```

Comment:
아래와 같이 기존 메서드를 사용하는 건 어떨까요? 🤔 

```suggestion
        if (!this.getSections().get(this.getLastSection().getDownStation().equals(station)) {
```

또한, `List<Section>` 을 [일급컬렉션](https://jojoldu.tistory.com/412) 으로 바꿔보시면 더 좋은 코드가 나올 것 같아요 👍 
[디미터 법칙](https://johngrib.github.io/wiki/law-of-demeter/)도 참고해주세요 😃

Reply:
앗 넵넵!! 참고하여 리팩토링 진행하겠습니다!